pkg/controllers/updaterun/execution.go

```diff
@@ -161,10 +164,17 @@ func (r *Reconciler) executeUpdatingStage(
 			markClusterUpdatingFailed(clusterStatus, updateRun.Generation, unexpectedErr.Error())
 			return 0, fmt.Errorf("%w: %s", errStagedUpdatedAborted, unexpectedErr.Error())
 		}
+
 		finished, updateErr := checkClusterUpdateResult(binding, clusterStatus, updatingStageStatus, updateRun)
 		if finished {
 			finishedClusterCount++
 			continue
+		} else {
+			// If cluster update has been running for more than 1 minute, mark the update run as stuck.
+			if time.Since(clusterStartedCond.LastTransitionTime.Time) > updateRunStuckThreshold {
+				emitUpdateRunStatusMetric(updateRun, updateRunMetricsStatusStuck)
+				fmt.Println("!!!threshold:", updateRunStuckThreshold)
```

Comment:
typo?

Please add a log instead

Reply:
oh, I was doing some local experiment and mistakenly pushed it. Removed. Thanks!